napari/layers/image/tests/test_pyramid.py

```diff
@@ -273,13 +273,13 @@ def test_metadata():
 
 def test_value():
     """Test getting the value of the data at the current coordinates."""
-    shapes = [(40, 20), (20, 10), (10, 5)]
+    shapes = [(40, 20), (20, 10)]
     np.random.seed(0)
     data = [np.random.random(s) for s in shapes]
     layer = Image(data, is_pyramid=True)
     value = layer.get_value()
     assert layer.coordinates == (0, 0)
-    assert value == (2, data[-1][0, 0])
+    assert value == (1, data[-1][0, 0])
```

Comment:
Why this change?

Reply:
I added the ability of the layer to "trim" the pyramid so that if you pass it say a pyramid that keeps going until the final layer is `1x1` the smallest resolution it would actually show is `63x63` (which is also the resolution it would use for the thumbnail). If all the layers are very small then the pyramid just takes the bottom two, which it is doing here. I've added a comment inside the test.